server/db/helper.js

```diff
@@ -50,14 +50,41 @@ const getCourses = () => Course.findAll({
 /**
  * Gets the user's profile info.
  * @param {integer} userId - The user's ID.
+ * @param {string} userEmail- The users's Email.
+ * @param {string} googleId- The users's google id
  */
-const getUser = (userId) => User.findOne({
+const getUser = (googleId) => User.findOne({
   where: {
-    id: userId,
+    googleId,
   },
-  attributes: { exclude: ['createdAt', 'updatedAt'] },
+  attributes: { exclude: ['createdAt', 'updatedAt', 'email', 'id'] },
 });
 
+/**
+ * Gets the user's profile info.
+ * @param {integer} userId - The user's ID.
+ * @param {string} userEmail- The users's Email.
+ * @param {string} givenName- The users's first name.
+ * @param {string} familyName- The users's last name.
+ * @param {integer} totalExperiencePoints- The users's total experience.
```

Comment:
We don't need to pass in a totalExperiencePoints value when creating a new user because they will always start at 0XP

Reply:
bet